src/Utilities/FileUtilities.cs

```diff
@@ -310,5 +311,28 @@ public static string PathRelativeTo(string path, string relativeToDirectory)
             Debug.Assert(string.Compare(Path.GetFullPath(Path.Combine(relativeToDirectory, returnVal)), fullPath, StringComparison.OrdinalIgnoreCase) == 0);
             return returnVal;
         }
+
+        /// <summary>
+        /// Returns the file name and extension of the specified path string.
+        /// Path.GetFileName will not trim the file name from a Windows path 
+        /// when running on Unix because `\` is a legal file char on Unix.
+        /// See the Remarks section in the docs: https://learn.microsoft.com/en-us/dotnet/api/system.io.path.getfilename
+        /// This method ignores that logic and provides the string after the last '\' when running on non-Windows platform and Path.GetFileName returns the same string.
+        /// </summary>
+        public static string GetPlatformIndependentFileName(string path)
+        {
+            var fileName = Path.GetFileName(path);
+#if !NETFRAMEWORK
```

Comment:
TraceEvent is only compiled for `netstandard20` now, so I don't think that this #if is necessary anymore.

Reply:
Yes, but this is a shared source file that is included in several other projects (e.g. PerfView.csproj) that compile for .NET Framework.